python/doc/usecases/use_case_fission_gas.rst

```diff
@@ -0,0 +1,67 @@
+.. _use-case-fission-gas:
+
+Fission gas release
+===================
+
+Fuel performance codes [1] like *TRANSURANUS* [2, 3] predict the behavior of nuclear fuel rods,
+such as the temperature, internal pressure, and fission gas release.
+This use case presents several simplified models of fission gas release,
+as a fraction of what is created by the fission reaction.
+They are derived from the work by [robertson2024]_.
+
+Each of the models has two inputs:
+
+- Single atom diffusion multiplier denoted by :math:`x_{\mathrm{diff}}`.
+- Fraction released due to micro-cracking, denoted by :math:`x_{\mathrm{crack}}`.
+
+Each model corresponds to different sets of experimental conditions.
+The number of different sets of environmental conditions is denoted by :math:`\sampleSize_{\mathrm{exp}}`,
+and in the following we use the index :math:`i` (:math:`1 \leq i \leq \sampleSize_{\mathrm{exp}}`)
+to identify a given set of environmental conditions.
+
+For each set of environmental conditions :math:`i`, fission gas release
+was measured, and the measured value is denoted by :math:`y_i`.
+The measured values are extracted from the International Fuel Performance Experiments (IFPE) database.
+Measurement uncertainty is represented by a normal distribution
+with mean :math:`y_i`, and its standard deviation :math:`\sigma_{y_i}` is a known function of :math:`y_i`:
+
+.. math::
+   \sigma_{y_i} = \sqrt{\left( \frac{y_i}{20} \right)^2 + 10^{-4}}
+
+The model corresponding to the environmental conditions :math:`i`
+is denoted by :math:`\model_i`.
+The values of the single atom diffusion multiplier :math:`x_{\mathrm{diff}, i}`
+and of the fraction of gas released due to micro-cracking :math:`x_{\mathrm{crack}, i}`
+corresponding to the measured :math:`y_i` are unobserved.
+
+The relationships between these quantities are represented in the following network.
+Full arrows represent deterministic relationships,
+while dashed arrows represent probabilistic relationships (e.g. a normal distribution).
```

Comment:
I do not think so in the context of this file (`use_case_fission_gas.rst`) where the only probabilistic relationship is

$$
g_i(x_{diff,i}, x_{crack,i}) \sim \mathcal{N}(y_i, \sigma_{y_i}).
$$

This distribution is not truncated, because it is the distribution of the model output. However, we *are* setting bounds on the input domain, so I probably must change some of the text in `plot_hierarchical_calibration_fission_gas.py`. Thanks for this remark!